test_pytest_easyread.py

```diff
@@ -135,3 +156,21 @@ def test_failing_function_no2():
         expected_result_1 = " \n\n1. failing function"
         expected_result_2 = ".\n\n1. failing function"
         assert expected_result_1 or expected_result_2 in result.stdout.str()
+
+    def test_pytest_easyread_works_with_parametrize(self, testdir):
```

Comment:
Great stuff - I wouldn't have thought to test this, so well done!

Reply:
Thank you 😊